wdio/tests/request-blocking.spec.ts

```diff
@@ -1,30 +1,31 @@
 import { usersResponseSelector, usersSelector, expectedURL } from '../data/test-data.json';
 import { expect } from 'chai';
+import { Mock } from 'webdriverio/build/types'
 
 describe('Request Blocking', () => {
   before(async () => {
     await browser.url('/');
   });
 
   describe('Blocking the request', () => {
-    let beforeActionResponse: string;
-    let afterActionResponse: string;
+    let mock: Mock;
 
     before(async () => {
       const getUsers = $(`${usersSelector}`);
       browser.setupInterceptor();
+      mock = await browser.mock(`**${expectedURL}`, { method: 'get' });
+      mock.abort('Aborted');
       await getUsers.click();
-      beforeActionResponse = await $(usersResponseSelector).getText();
+    });
 
-      const mockResponse = await browser.mock(`**${expectedURL}`, { method: 'get' });
-      mockResponse.abort('Aborted');
-      await getUsers.click();
-      afterActionResponse = await $(usersResponseSelector).getText();
+    it("the request should fail", () => {
+      const errorReason = (mock as any).respondOverwrites[0].errorReason;
```

Comment:
In the 11th line you declare `mock: Mock`. Why is it casted to any in this line?

Reply:
Unfortunately, I get the following error if I do not case it to any:
_Property 'respondOverwrites' does not exist on type 'Mock'._
I do not really know why, because this property exists, I also checked it by logging it out.